### PR TITLE
fix buttons and font size, fix select

### PR DESCRIPTION
**Pull request by andreabiagi**
size and class of buttons fixed!
I still have to check the classes of buttons in all html files.

Buttons.html files added. Now you can check fonts and buttons size. 
Some icons (like eyes) result a little bit small. We have to resize some svg file! 

Buttons with icons added in buttons.html.

Select added in buttons.html.

Tell me if you find some missing icon before exporting the new font.

### CODE REVIEW

**Comment by Gnafu** [comment]
Why the buttons are now white with the blue border ?

**Comment by andreabiagi** [comment]
Grey was to sad! Now buttons are cooler =)

File: theme/default/css/css.css [hunk 1]
```diff
@@ -2048,11 +2048,13 @@ fieldset[disabled] .btn {
...
   box-shadow: none;
 }
+
 .btn-default {
   color: #078aa3;
-  background-color: #eeeeee;
   border:none;
   font-size:28px;
+  background-color:#ffffff;
+  border:#078aa3 1px solid;
```
Comment (Gnafu) [inline theme/default/css/css.css:2057]: duplicated border in style

**Comment by Gnafu** [comment]
Merged

**Pull request closed without merging**